ForumProject/projects/admin.py

```diff
@@ -1,3 +1,46 @@
 from django.contrib import admin
+from .models import Project, ProjectFiles, InvestorProject, ProjectLog
 
-# Register your models here.
+class ProjectAdmin(admin.ModelAdmin):
+    list_display = ['id',
+                    'name',
+                    'startup',
+                    'description',
+                    'status',
+                    'created_at',
+                    'updated_at',
+                    'duration',
+                    'budget_currency',
+                    'budget_amount'
+                    ]
+    search_fields = ['name', 'description']
+
+admin.site.register(Project, ProjectAdmin)
+
+class ProjectFilesAdmin(admin.ModelAdmin):
+    list_display = ['id', 'project', 'file_description', 'file']
+    search_fields = ['id', 'project', 'file_description', 'file']
+
+admin.site.register(ProjectFiles, ProjectFilesAdmin)
+
+class InvestorProjectAdmin(admin.ModelAdmin):
+    list_display = ['id', 'investor', 'project', 'share']
```

Comment:
You might want to include fields from the related `Investor` and `Project` models in `list_display` by defining a method in the `admin` class that fetches the specific related field you want to display

Reply:
I consider this is not necessary, for admin purposes it is enough to see `__str__` representation of `Investor` and `Project`